### PR TITLE
Show sonar altitude, if sensor exist

### PR DESCRIPTION
**Pull request by HThuren**
Show Sonar altitude at Setup tab, if Sonar respond from active Sensor.
image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: src/js/msp/MSPCodes.js
Comment (haslinghuis) [inline src/js/msp/MSPCodes.js:38]: This assumes the sonar is placed downwards. We need to prepare for the possibility of object detection and avoidance using this sensor. So I think we need to leave this as is and rename in firmware when we start improving the sonar code.

Comment (HThuren) [reply src/js/msp/MSPCodes.js:38]: Right, no isuse, I recover

**Review by haslinghuis — APPROVED** [review]

**Comment by nerdCopter** [comment]
approving untested.

**Comment by ibrahimqazi** [comment]
> Show Sonar altitude at Setup tab, if Sonar respond from active Sensor. image

Hello @HThuren , does this also display LiDAR range-finder data or only sonar sensor ?

**Comment by HThuren** [comment]
Hi https://github.com/ibrahimqazi , LIDAR are covered and attitude are supposed to be there. But at time for development I did not have a LiDAR sensor, so for sure.

**Pull request merged**
